tls/ttls/net.h

```diff
@@ -134,34 +137,6 @@ int mbedtls_net_accept( mbedtls_net_context *bind_ctx,
                         void *client_ip, size_t buf_size, size_t *ip_len );
 
 /**
- * \brief          Set the socket blocking
- *
- * \param ctx      Socket to set
- *
- * \return         0 if successful, or a non-zero error code
- */
-int mbedtls_net_set_block( mbedtls_net_context *ctx );
```

Comment:
Well, when you remove some symbol, please check whether there is some of it users. Now I see

         $ find . -name \*.[ch] |xargs grep --color mbedtls_net_set_block
         ./ttls/compat-1.3.h:#define net_set_block mbedtls_net_set_block
         ./programs/ssl/ssl_server2.c:        ret = mbedtls_net_set_block( &client_fd );
         ./programs/ssl/ssl_client2.c:        ret = mbedtls_net_set_block( &server_fd );
         ./programs/ssl/ssl_client2.c:            ret = mbedtls_net_set_block( &server_fd );

```/programs/ssl/*``` are likely to be adjusted in near future, so leave them. While ```./ttls/compat-1.3.h``` seems like a header for compatibility for programs written for old versions of mbedtls, so please remove it.